docs/resources/fgs_function.md

```diff
@@ -53,51 +53,151 @@ EOF
-	set to jar or zip. Changing this creates a new function.
+* `depend_list` - (Optional, String) Dependencies of the function.
 
-* `handler` - (Required, String, ForceNew) Entry point of the function. Changing this creates a new function.
+* `user_data` - (Optional, String) Key/Value information defined for the function.
 
-* `memory_size` - (Required, Int, ForceNew) Memory size(MB) allocated to the function. Changing this creates a new function.
+* `agency` - (Optional, String) This parameter is mandatory if the function needs to access other cloud services.
 
-* `runtime` - (Required, String, ForceNew) Environment for executing the function. Changing this creates a new function.
+* `app_xrole` - (Optional, String) This parameter is mandatory if the function needs to access apps in other cloud services.
 
-* `timeout` - (Required, Int, ForceNew) Timeout interval of the function, ranges from 3s to 900s. Changing this creates a new function.
+* `description` - (Optional, String) Description of the function.
 
-* `user_data` - (Optional, String, ForceNew) Key/Value information defined for the function. Changing this creates a new function.
+* `initializer_handler` - (Optional, String) Initializer of the function.
 
-* `agency` - (Optional, String, ForceNew) This parameter is mandatory if the function needs to access other cloud services.
-	Changing this creates a new function.
+* `initializer_timeout` - (Optional, Int) Maximum duration the function can be initialized. Value range: 1s to 300s.
 
-* `func_code` - (Required, String, ForceNew) Function code. When code_type is set to inline, zip, or jar, this parameter is mandatory,
-	and the code can be encoded using Base64 or just with the text code. Changing this creates a new function.
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project id of the function.
+  Changing this creates a new function.
 
+* `func_vpc` - (Optional, List) VPC configuration. The `func_vpc` object structure is documented below.
+
+* `mount_config` - (Optional, List) File system configuration. The `mount_config` object structure is documented below.
+
+
+The `func_vpc` block supports:
+
+* `vpc_id`  - (Required, String) The ID of VPC.
+
+* `subnet_id`  - (Required, String) The ID of subnet.
+
+The `mount_config` block supports:
+
+* `mount_user` - (Optional, List) File system user configuration.
+  * `user_id` - (Optional, String) User ID, a non-0 integer from –1 to 65534. Default to -1.
+  * `user_group_id` - (Optional, String) User group ID, a non-0 integer from –1 to 65534. Default to -1.
+
+* `func_mounts` - (Optional, List) File system list.
+  * `mount_type` - (Required, String) Mount type. Options: sfs, sfsTurbo, and ecs.
+  * `mount_resource` - (Required, String) ID of the mounted resource (corresponding cloud service).
+  * `mount_share_path` - (Required, String) Remote mount path. Example: 192.168.0.12:/data.
+  * `local_mount_path` - (Required, String) Function access path.
```

Comment:
suggest to update the description and structure of list parameters.

Reply:
fixed